project/RootSettings.scala

```diff
@@ -5,18 +5,10 @@ package akka
 
 import sbt._
 import sbt.Keys._
-import akka.ValidatePullRequest.validatePullRequest
-import sbtunidoc.Plugin.UnidocKeys.unidoc
-import com.typesafe.sbt.site.SphinxSupport
-import com.typesafe.sbt.site.SphinxSupport.Sphinx
 
 object RootSettings extends AutoPlugin {
 
-  lazy val docs = ProjectRef(file("."), "akka-docs")
-
   // settings for root project goes here
   override val projectSettings = Seq(
-    validatePullRequest <<= validatePullRequest.dependsOn(unidoc in Compile),
-    validatePullRequest <<= validatePullRequest.dependsOn(SphinxSupport.generate in Sphinx in docs)
   )
 }
```

Comment:
After a bit of though I agree with removing `RootSettings` autoplugin and adding other plugins explicitly, mainly because this is how we do it in sub projects. Root project should not be treated differently.